pkg/authorization/api/register.go

```diff
@@ -89,16 +88,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 
 		&RoleBindingRestriction{},
 		&RoleBindingRestrictionList{},
-	}
-	scheme.AddKnownTypes(SchemeGroupVersion,
-		append(types,
-			&metav1.Status{}, // TODO: revisit in 1.6 when Status is actually registered as unversioned
```

Comment:
status isn't included in metav1.AddToGroupVersion... @smarterclayton should it be?